CRM/Mailing/BAO/Mailing.php

```diff
@@ -924,46 +924,6 @@ public function getVerpAndUrlsAndHeaders($job_id, $event_queue_id, $hash) {
     return [&$verp, &$urls, &$headers];
   }
 
-  /**
-   * Return a list of group names for this mailing.  Does not work with
-   * prior-mailing targets.
-   *
-   * @deprecated since 5.71 will be removed around 5.77
```

Comment:
Looking for callers, I see:

*  `getGroupNames` called by `CRM_Utils_Token::getMailingTokenReplacement` 
* `getMailingTokenReplacement` called by `CRM_Mailing_Tokens::evaluateToken()` and `CRM_Utils_Token::replaceMailingTokens()` 
* `replaceMailingTokens()` called by `CRM_SMS_Form_Upload` and `CRM_Mailing_Event_BAO_MailingEventResubscribe`

It might be that...
* The code-paths are dead for other/higher-level reason (*in which case more can be cleaned*), or....
* These specific callers lack test-coverage for the `{mailing.group}` token.

Reply:
This deprecation was suggested in f31f94b22849226e49f08af003c7ab59e3de3ca9. @eileenmcnaughton, does this actually look safe to remove?